mm2src/docker_tests/qrc20_tests.rs

```diff
@@ -1388,7 +1388,11 @@ fn test_get_max_taker_vol_and_trade_with_dynamic_trade_fee(coin: QtumCoin, priv_
         .expect("!send_taker_payment");
 
     let my_balance = coin.my_spendable_balance().wait().expect("!my_balance");
-    assert_eq!(my_balance, 0.into());
+    assert_eq!(
```

Comment:
How often does this assertion fail?

Reply:
As discussed in person it fails not that often, but we might possibly make it even better: https://github.com/KomodoPlatform/atomicDEX-API/issues/891